CONTRIBUTING.md

```diff
@@ -61,13 +61,9 @@ that are published to the repository.  We use environment variables to store
 configuration data.  This makes it possible to easily change between
 environments of a deployed application (see [The Twelve-Factor
 App](https://www.12factor.net/config)).  The .env file you create here is pre-
-configured to be ignored by the repo, so do not explicitly check it in.
+configured to be ignored by the repository, so do not explicitly check it in.
 
-Copy the NeuroSimulator environment example:
-
-    cp NeuroSimulator/.env.example NeuroSimulator.env
-
-Update the .env file to reflect the new secret key.
+Update the .env file to reflect the new secret key with a line start with "SECRET_KEY="
```

Comment:
I suggest we simply use an empty key (or something that doesn't look like a key like `CHANGETHISVALUE`) and keep the the example file.